corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -140,6 +140,30 @@ def input_trans(name, langs=None, input_name='name', input_id=None, data_bind=No
     return format_html(template, **options)
 
 
+@register.simple_tag
+def textarea_trans(name, langs=None, input_name='name', input_id=None, data_bind=None):
```

Comment:
There's a fair amount of duplication with `input_trans`. Thoughts on instead making the node name a param? (Which admittedly makes the function name make less sense.)

Reply:
> Thoughts on instead making the element name a param? 

You'd need to update other places where it's used by pass in the new element name param, "input" in current usages, and also account for `type` of input, which isn't a consideration for `text area`.

Reply:
It could default to `"input"`, but yeah, fair point that `type` isn't relevant for textareas.

Reply:
I have conflicting thoughts - I think it makes sense to combine from the side of them being text entries with the same basic functionality, and almost did so. At the same time, when I think about the `<input>` element, it's used for so many things that aren't just `input type="text"` that I would hesitate to include the `<textarea>` element in the same function.